wazo_calld_queue/bus_consume.py

```diff
@@ -105,7 +151,42 @@ def _queue_member_status(self, event):
             required_acl='events.queues'
         )
         bus_event.routing_key = 'calls.queues.member_status'
-        self.bus_publisher.publish(bus_event)
+        tenant_uuid, user_uuids = self._extract_tenant_from_event(event)
+        headers = self._build_headers(
+            user_uuids=user_uuids,
+            tenant_uuid=tenant_uuid,
+        )
+        self.bus_publisher.publish(bus_event, headers=headers)
 
     def _publish(self, bus_event):
         self.bus_publisher.publish(bus_event, headers={'name': bus_event.name})
+
+    def _extract_tenant_from_event(self, event):
```

Comment:
ouch, we need to think to a better solution, if it work for you it's great, this is clearly not possible to have to do that for building something on the top of Wazo Platform. Performance will also probably decrease with this function.